api/views/submission.py

```diff
@@ -377,28 +378,27 @@ def done(self, request: Request, pk: int, username: str = None) -> Response:
 
         if submission.claimed_by is None:
             return Response(status=status.HTTP_412_PRECONDITION_FAILED)
-
         mod_override = (
             request.data.get("mod_override", False) and request.user.is_grafeas_staff
         )
 
         if not mod_override:
             if submission.claimed_by != user:
                 return Response(status=status.HTTP_412_PRECONDITION_FAILED)
+            transcription = Transcription.objects.filter(submission=submission).first()
+            if not transcription:
+                return Response(status=status.HTTP_428_PRECONDITION_REQUIRED)
+            self._check_for_rank_up(user, submission)
```

Comment:
@itsthejoker Shouldn't this check still be outside of the mod override? Since the user is still awarded gamma?

Reply:
You're right, I'll fix that